babel.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+	ignore: ['node_modules/detox'],
```

Comment:
this line is used to prevent the e2e injections introducing detox into babel compiling, with this ignore, there will be an error if any app code rely on detox.